packages/flutter_app/pubspec.yaml

```
@@ -55,17 +55,17 @@ dependencies:
 dev_dependencies:
   altive_lints: ^1.6.1
   build_runner: ^2.3.3
-  custom_lint: ^0.2.12
+  custom_lint: ^0.3.4
   flutter_gen_runner: ^5.1.0+1
-  flutter_launcher_icons: ^0.10.0
+  flutter_launcher_icons: ^0.13.1
   flutter_test:
     sdk: flutter
   freezed: ^2.1.1
   go_router_builder: ^1.0.16
   json_serializable: ^6.6.1
   mockito: ^5.2.0
-  riverpod_generator: ^2.0.0
-  riverpod_lint: ^1.1.2
+  riverpod_generator: ^2.2.0
```

こちらも🙏
https://pub.dev/packages/riverpod_generator/changelog

```suggestion
  riverpod_generator: ^2.2.1
```